modules/datastore/src/EventSubscriber/DatastoreSubscriber.php

```diff
@@ -147,4 +148,26 @@ public function drop(Event $event) {
     }
   }
 
+  /**
+   * React to a preReference to check if datastore update should be triggered.
+   *
+   * @param \Drupal\common\Events\Event $event
+   *   The event object containing the resource uuid.
+   */
+  public function onPreReference(Event $event) {
+    $data = $event->getData();
+    $original = $data->getOriginal();
+    $field = \Drupal::service('config.factory')->getEditable('datastore.settings')->get('triggering_property');
+    $field = $field ? $field : 'modified';
```

Comment:
Let's avoid hard-coding "modified" here. I know the issue suggested otherwise but realizing this should just default to nothing.